### PR TITLE
Missed rules

### PR DESCRIPTION
**Pull request by antares1470**
### Description of the Change

Fixed up some of format string sonar smells that were missed the previous time a PR was done for this

### CODE REVIEW

File: CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/infomap/InfomapBase.java [hunk 1]
```diff
@@ -761,7 +761,7 @@ private void partition(final int recursiveCount, final boolean fast, final boole
         if (verbose) {
             if (config.getVerbosity() == 0) {
                 final String fmt = String.format("%s.%df", "%", config.getVerboseNumberPrecision());
-                System.out.printf("to %d modules with codelength " + fmt + "%n", getNumTopModules(), codelength);
+                System.out.printf("to %d modules with codelength %s%n", getNumTopModules(), fmt, codelength);
```
Comment (antares1470) [inline CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/infomap/InfomapBase.java:764]: This is the one I'm unsure about. Not sure whether I actually fixed this properly. `codelength` threw me out

Comment (arcturus2) [reply CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/infomap/InfomapBase.java:764]: I'd agree. It looks like the wrong number of args.

File: CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/infomap/InfomapBase.java [hunk 2]
```diff
@@ -761,7 +761,8 @@ private void partition(final int recursiveCount, final boolean fast, final boole
         if (verbose) {
             if (config.getVerbosity() == 0) {
                 final String fmt = String.format("%s.%df", "%", config.getVerboseNumberPrecision());
-                System.out.printf("to %d modules with codelength " + fmt + "%n", getNumTopModules(), codelength);
+                final String cl = String.format(fmt, codelength);
+                System.out.printf("to %d modules with codelength %s%n", getNumTopModules(), cl);
```
Comment (antares1470) [inline CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/infomap/InfomapBase.java:765]: Hopefully this one is alright. I don't think it's the cleanest solution but I'm struggling to come up with a better one right now

Comment (arcturus2) [reply CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/infomap/InfomapBase.java:765]: This was what I was thinking. Let me know if I've missed the mark:

`System.out.printf("to %d modules with codelength %s.%df%n", getNumTopModules(), "%", config.getVerboseNumberPrecision());`

Comment (antares1470) [reply CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/infomap/InfomapBase.java:765]: The issue is that it leaves out the codelength variable. %s.%df is only used to get the correct formatter for codelength and therefore I believe there needs to an additional step to apply codelength to the given format

**Pull request merged**